src/core/Recoil_Batching.js

```diff
@@ -10,9 +10,14 @@
  */
 
 const {batchStart} = require('../core/Recoil_RecoilValueInterface');
-const {unstable_batchedUpdates} = require('../util/Recoil_ReactBatchedUpdates');
 
-let batcher = unstable_batchedUpdates;
+let batcher = function (_func) {
+  throw new Error('Unexpected Fatal Error: undefined batcher function');
+};
+// for testing, use default batcher defined in setupJestMock.js
+if (global && global._jestDefaultBatcher) {
+  batcher = global._jestDefaultBatcher;
```

Comment:
Could we avoid `setupJestMock.js` and inside the default batcher just check if running under Jest and use the `unstable_batchedUpdates()` if so, otherwise throw the error?  Or maybe just call `Recoil.setBatcher()` in some Jest setup function?